documentation/guides/general_setup.md

```diff
@@ -200,6 +200,13 @@ different editor if you have a preference.
 
 The following setup steps are only needed in very specific scenarios.
 
+### coreutils (only for macOS)
+
+`coreutils` adds GNU utils to macOS. `timeout` from the package is required. 
+You can install it using Homebrew [here](https://formulae.brew.sh/formula/coreutils).
+Homebrew is a [package manager](https://en.wikipedia.org/wiki/Package_manager) for macOS 
+which can be downloaded [here](https://brew.sh).
```

Comment:
In general, 'here' breaks the flow of a sentence. You can rephrase it like so:

```markdown
You can install the 
[`coreutils` formula](https://formulae.brew.sh/formula/coreutils) using 
[Homebrew](https://brew.sh), which is a package manager for macOS.
```

Reply:
Good point, I'll update the wording.